lib/teleterm/api/proto/v1/auth_settings.proto

```diff
@@ -32,6 +32,12 @@ message AuthSettings {
     // banner text that must be retrieved, displayed and acknowledged by
     // the user.
     bool has_message_of_the_day = 5;
+    // auth_type is the authentication type e.g. "local", "github", "saml", "oidc"
```

Comment:
Thanks for the examples, much better.

Should we call this second_factor_type, so it matches the lingo in other parts of the project? (https://cs.github.com/gravitational/teleport/blob/83256ed7600e7ce20e899830f68eecfb2ae523a3/api/constants/constants.go#L146)

Reply:
hm, i'm probably misunderstanding, but we do have a field [second_factor](https://github.com/gravitational/teleport/blob/7ca79d4516656810394dcea9b7c4f512b487a569/lib/teleterm/api/proto/v1/auth_settings.proto#L25) already for this message (which values are "on, optional, off")? this configuration is also pretty similar to our [web config](https://github.com/gravitational/teleport/blob/7ca79d4516656810394dcea9b7c4f512b487a569/api/client/webclient/webconfig.go#L67) as well

Reply:
Nevermind, you are correct.